packages/file_selector/file_selector_windows/windows/test/file_selector_plugin_test.cpp

```diff
@@ -341,17 +322,78 @@ TEST(FileSelectorPlugin, TestSaveWithArguments) {
   std::string initial_directory("C:\\Program Files");
   std::string suggested_name("a name");
   std::string confirm_button("Save it!");
-  ErrorOr<EncodableList> result = plugin.ShowSaveDialog(
-      CreateOptions(AllowMultipleArg(false), SelectFoldersArg(false),
-                    EncodableList()),
+  ErrorOr<FileDialogResult> result = plugin.ShowSaveDialog(
+      SelectionOptions(/* allow multiple = */ false,
+                       /* select folders = */ false, EncodableList()),
       &initial_directory, &suggested_name, &confirm_button);
 
   EXPECT_TRUE(shown);
   ASSERT_FALSE(result.has_error());
-  const EncodableList& paths = result.value();
+  const EncodableList& paths = result.value().paths();
+  EXPECT_EQ(paths.size(), 1);
```

Comment:
Given the next line, arguably `ASSERT_EQ`.

Reply:
Fixed everywhere.